auth/password-visibility.js

```diff
@@ -0,0 +1,15 @@
+const visible = document.getElementsByClassName('visible');
```

Comment:
💊 제안
변수나 클래스명, 이미지명과 같은 이름은 해당 파일이 어떤 역할을 하는 친구인지 나타내는 큰 단서가 되고 코드파악 및 유지보수에 중요한 역할을 합니다. 따라서 가능한 자세한 이름을 짓는 것이 좋아요.
지금 visible이라고 명명한 변수는 자세한 정보를 전달하지 못하니 아래처럼 더 자세히 적는 것을 추천드려요(아래는 이러한 얘기를 하구나 수준으로 참고만해주세요~)

```suggestion
const passwordToggleButton = document.getElementsByClassName('visible');
```